packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -69,10 +69,10 @@ class RouteConfig {
   ) {
     assert(!reader.isNull, 'reader should not be null');
     final InterfaceType type = reader.objectValue.type! as InterfaceType;
-    // Ignore the deprected `element2` so that the "downgraded_analyze" CI step
+    // Ignore the deprecated `element2` so that the "downgraded_analyze" CI step
```

Comment:
I removed the comment in [f0924ed](https://github.com/flutter/packages/pull/3439/commits/f0924edc8da7090855bcd33a646c3e91f5d2bad6) since it no longer applies.